src/order/rel_iso.lean

```diff
@@ -148,6 +148,17 @@ theorem preimage_equivalence {α β} (f : α → β) {s : β → β → Prop}
 
 namespace rel_embedding
 
+/--
+To define an relation embedding from an antisymmetric relation `r` to a reflexive relation `s` it
+suffices to give a function together with a proof that it satisfies `s (f a) (f b) ↔ r a b`.
+-/
+@[simps]
```

Comment:
Does `@[simps]` automatically add an `of_map_rel_iff_apply`-type lemma?

Reply:
It makes a lemma `⇑(of_map_rel_iff f hf).to_embedding x = f x` - perhaps it'd be helpful to have a `coe_to_fun` version though, I'll add that